Library/src/Web/EmscriptenWebClient/EmscriptenWebClient.cpp

```diff
@@ -70,10 +70,26 @@ void OnFetchSuccessOrError(emscripten_fetch_t* Fetch)
 
 	for (int i = 0; i < Keys->Size(); ++i)
 	{
-		csp::common::String Key = Keys->operator[](i);
-		csp::common::String Val = Headers[Key];
-
-		Payload.AddHeader(Key, Val);
+		auto Key	  = Keys->operator[](i);
+		auto Val	  = std::string(Headers[Key].c_str());
+		auto KeyValue = std::string(Key.c_str());
+		// Make Key and Val lower-case
+		std::transform(KeyValue.begin(),
+					   KeyValue.end(),
+					   KeyValue.begin(),
+					   [](unsigned char c)
+					   {
+						   return std::tolower(c);
+					   });
+		std::transform(Val.begin(),
```

Comment:
We should not be touching the value at all. Only `Key` should be transformed.

Reply:
After discussion we agreed that since POCOWebClient already transforms the values, we'll also do that here as well.